bot/cogs/source.py

```diff
@@ -0,0 +1,99 @@
+import inspect
+import os
+from typing import Union
+
+from discord import Embed
+from discord.ext.commands import BadArgument, Cog, Command, Context, Converter, HelpCommand, command
```

Comment:
This is getting "dangerously" close to a lot of imports, but you don't _have_ to change this. I just want to let you know for the future that we're trying to get away from such long imports, preferring to qualify the names instead. If this only had 2 or 3 things then it wouldn't really matter.